EIPS/eip-4844.md

```diff
@@ -210,10 +210,10 @@ def point_evaluation_precompile(input: Bytes) -> Bytes:
     # Verify commitment matches versioned_hash
     assert kzg_to_versioned_hash(commitment) == versioned_hash
 
-    # Verify KZG proof
+    # Verify KZG proof with z and y in big endian format
     assert verify_kzg_proof(commitment, z, y, proof)
 
-    # Return FIELD_ELEMENTS_PER_BLOB and BLS_MODULUS as padded 32 byte big endian values
+    # Return FIELD_ELEMENTS_PER_BLOB and BLS_MODULUS as padded 32 bytes big endian values
```

Comment:
I think the previous version is the correct one (it's "a 32 byte value", not "a 32 bytes value").

Reply:
thank you! updated :+1: